packaging/hudi-cli-bundle/pom.xml

```diff
@@ -298,5 +299,15 @@
       <artifactId>log4j-core</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
```

Comment:
have you also considered the kafka-connect, timeline server and metaserver bundles?

Reply:
Yes, `timeline server and metaserver` already includes the slf4j jar in current code base, there is no need to change.
for `kafka-connect`, similar with other engines like Spark and Flink, we should not include the slf4j, like it does not incude the log4j too before, the kafka should suppliy the logger jar in the classpath.